# goal
string location_name
float64[] coordinate
string[] options

---
# result
bool result

---
# feedback
string data
